src/index.ts

```diff
@@ -81,6 +82,8 @@ const defaultVariant: IVariant = {
 };
 const storeKey = 'repo';
 
+type SdkState = 'initializing' | 'healthy' | 'error';
```

Comment:
Just another small detail: If I see my SDK state is error, should I trust its results? Maybe `stale` is more accurate to how our SDK behaves: if some request fails, we still respond correctly with potentially stale data.

Reply:
It's a private property though, so we only use it internally in the SDK to discern whether or not we should emit the recovered event. In this case I think it's actually more accurate for the reader to use 'error' and it's not exposed anywhere outside of the internals of the SDK.